components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.store.ui/src/main/java/org/wso2/carbon/mediator/store/MessageStoreMediator.java

```diff
@@ -43,6 +48,8 @@
 
     private String messageStoreName;
 
+    private SynapsePath messageStoreExp;

```

Comment:
what is a messageStoreExp? Shall we add a description.

Reply:
Done